### PR TITLE
Add localization for accessibility labels

### PR DESCRIPTION
**Pull request by igorkravchenko**
Add localized strings for VoiceOver.

### CODE REVIEW

**Comment by kuldar-daniel** [comment]
@dukhovnyi @igorkravchenko the issue with the approach is that it will not be backwards compatible. Meaning, if existing integrators have defined their own styles for specific views, and now update to where accessibility is required argument, they will face errors with the new sdk

**Comment by igorkravchenko** [comment]
> @dukhovnyi @igorkravchenko the issue with the approach is that it will not be backwards compatible. Meaning, if existing integrators have defined their own styles for specific views, and now update to where accessibility is required argument, they will face errors with the new sdk

@kuldar-daniel I think this is a good point. Do you have ideas on how this can be done differently to preserve backward compatibility and still have ability to localize accessibility labels and hints?

**Comment by kuldar-daniel** [comment]
@igorkravchenko I think we should provide default values in the initialisers

**Comment by igorkravchenko** [comment]
> @igorkravchenko I think we should provide default values in the initialisers

@kuldar-daniel After discussing this point with @dukhovnyi, it seems like it would be necessary to introduce breaking changes. Integrators will need to provide `Accessibility` for every style that requires it. However we added `Accessibility.unsupported` constant to allow integrators to bypass providing concrete strings, using the constant. This will make integrator aware that accessibility properties can be added, but they also can be omitted. But it is on integrator to make the choice.

File: GliaWidgets/Component/Bubble/BubbleStyle.Accessibility.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
+extension BubbleStyle {
+    /// Accessibility properties for BubbleStyle.
+    public struct Accessibility: Equatable {
```
Comment (kuldar-daniel) [inline GliaWidgets/Component/Bubble/BubbleStyle.Accessibility.swift:3]: Why do they need to conform to Equatable protocol?

Comment (kuldar-daniel) [inline GliaWidgets/Component/Bubble/BubbleStyle.Accessibility.swift:3]: Why do they need to conform to Equatable protocol?

**Review by kuldar-daniel — APPROVED** [review]

File: GliaWidgets/Component/Bubble/BubbleStyle.Accessibility.swift (hunk 1 above)
Comment (igorkravchenko) [reply GliaWidgets/Component/Bubble/BubbleStyle.Accessibility.swift:3]: I plan to add unit tests that make sure that every style that has `Accessibility` can have `Accessibility.unsupported` passed to it. So basically for XCTAssertEqual.

Comment (igorkravchenko) [reply GliaWidgets/Component/Bubble/BubbleStyle.Accessibility.swift:3]: [For XCTAssertEqual](https://github.com/salemove/ios-sdk-widgets/pull/253#discussion_r845776310).

**Review by dukhovnyi — APPROVED** [review]

**Pull request merged**